articles/azure-monitor/app/website-monitoring.md

```diff
@@ -43,7 +43,7 @@ Application Insights can gather telemetry data from any internet-connected appli
     | Settings        | Value           | Description  |
    | ------------- |:-------------|:-----|
    | **Name**      | Globally Unique Value | Name that identifies the app you are monitoring |
-   | **Resource Group**     | myResourceGroup      | Name for the new resource group to host App Insights data |
+   | **Resource Group**     | myResourceGroup      | Name for group o the new resource to host App Insights data |
```

Comment:
@bergano65 how it is currently formulated to create/delete a resource group is by design and a pattern we use across Azure Docs. While it is certainly possible to use an existing resource group and then only have instructions for deleting the individual resource we found that for more complex quickstarts there are often scenarios where multiple resources are created and new customers would sometimes accidentally only delete a subset of the resources and then get hit with a bill for resources they weren't using at the end of the month. 

By steering brand new users towards creating a new resource group that is dedicated to their quickstart experiments, it makes the process of deleting resources much easier since they just have to delete the resource group, and all the child resources get deleted along with it. Again this isn't as much of an issue with an App Insights quickstart where you might only be creating a single resource, but where possible we try to create a consistent experience across quickstarts on Azure Docs. 

If your concern is that someone who doesn't follow the instructions and uses an existing resource group instead of creating a new one could potentially delete other resources in the process I'm happy to add an additional note of caution to that section to remind a reader that the instructions only apply to if you have created a new resource group. 

As far as the line level change you have made, in this case I think the original may still be more appropriate. While I agree that the repetition of _resource group_ may feel a little redundant a _resource group_ is a distinct entity within azure, and there are other types of groups within Azure so I don't think it necessarily hurts to reiterate that in the table.     

This table is also mirrored across the App Insights quickstarts so if we were going to change one I would want to change all of them at the same time.

Is there something idiomatic, or from a localization standpoint that is problematic about the current construction of that sentence? Or is it just that it feels redundant/unnecessary?

Reply:
1. my thought was that user my understood  erroneously that for new resource he should create new group always. probably mention that's he has choice use existing or new group . but we recommend use new group for simplicity of starting.
2.1 about cleanup: probably mention that delete group delete child resources also.

Reply:
and about warning about unexpected delete  existing group  results: from one side agree it's  correct but same time your thoughts start should be as simple as possible and too many details is bad.
so decide yourself

Reply:
@bergano65 thanks for the feedback. I have added a line indicating that you can use an existing resource group or create a new one as well as added further clarification to the section on deleting resource groups. These changes are merged internally and will go-live tomorrow.